scripts/build/AppImages/desktopintegration

```diff
@@ -69,13 +111,13 @@ yesno()
   TITLE=$1
   TEXT=$2
   if [ -x /usr/bin/zenity ] ; then
-    LD_LIBRARY_PATH="" zenity --question --title="$TITLE" --text="$TEXT" || exit 0
+    LD_LIBRARY_PATH="" zenity --question --title="$TITLE" --text="$TEXT" 2>/dev/null || exit 0
   elif [ -x /usr/bin/kdialog ] ; then
     LD_LIBRARY_PATH="" kdialog --caption "Disk auswerfen?" --title "$TITLE" -yesno "$TEXT" || exit 0
```

Comment:
@lurch Hm, that is weird, I never noticed it. This script comes without any modifications from the AppImageKit project, so I suggest raising an issue at https://github.com/probonopd/AppImageKit, and I'll update accordingly afterwards.